config/install-kind/namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
I think this should be included by default in all installation not just kind?

Reply:
I have done this in my PR/branch for now as well: https://github.com/substratusai/substratus/pull/185/commits/01d2ec10fd41dbf126348534888411eb0f43c1b6

Approach I took is add namespace.yaml under config/manager. I thought it wasn't worth creating a separate directory.